model/reactions.tsv

```diff
@@ -6024,11 +6024,11 @@ rxns	rxnKEGGID	rxnBiGGID	rxnEHMNID	rxnHepatoNET1ID	rxnREACTOMEID	rxnRecon3DID	rx
 "HMR_5984"	""	""	""	"r2069"	""	"r2069"	"MNXR105978"	"HMR_5984"	"RCR40526"	"2.A.3.8.15"	0	"MA-R05984"	""	""
 "HMR_5985"	""	""	""	"r2070"	""	"r2070"	"MNXR105979"	"HMR_5985"	"RCR40160"	"2.A.3.8.15"	0	"MA-R05985"	""	""
 "HMR_5986"	""	""	""	"r2071"	""	"r2071"	"MNXR105980"	"HMR_5986"	"RCR41449"	"2.A.3.8.15"	0	"MA-R05986"	""	""
-"HMR_5987"	""	"FE2t"	""	"r2072"	""	"FE2DMT1"	"MNXR99505"	"HMR_5987"	"RCR40527"	"2.A.55.2.3"	0	"MA-R05987"	""	""
+"HMR_5987"	""	"FE2t"	""	"r2072"	""	"FE2DMT1"	"MNXR99505"	"HMR_5987"	"RCR40527"	"2.A.55.2.3"	0	"MA-R05987"	"RHEA:29441;RHEA:29440;RHEA:29581;RHEA:29580"	"RHEA:29439;RHEA:29579"

```

Comment:
Here, the Rhea master IDs can be difficult to associate programmatically to the Rhea IDs (2 master IDs versus to 4 Rhea IDs). Do you think the content should be changed, for instance by always having the same number of IDs in the two columns (this implies to duplicate Rhea master IDs)?

Reply:
Another solution would be to manually curate the association now, some Rhea ID can be probably removed.

Reply:
Among the four Rhea ids mapped to `HMR_5987`, only `RHEA:29581` appears to be the correct association while the others can be ruled out according the reaction direction or metabolite compartment.